src/main.jsx

```diff
@@ -10,24 +10,11 @@ import thunkMiddleware from 'redux-thunk'
 import createLogger from 'redux-logger'
 import { Router, hashHistory } from 'react-router'
 import cozy from 'cozy-client-js'
```

Comment:
We probably need to remove this dep as the `cozy-client-js` asset is now injected by the template to stuck the last compatible version with the stack, nope?